backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/GraphqlApi.java

```diff
@@ -66,8 +66,8 @@ public static String handleSchemaRequests(Request request, Response response) th
     MolgenisSession session = sessionManager.getSession(request);
     String schemaName = sanitize(request.params(SCHEMA));
 
-    // apps is not a schema but a resource
-    if ("apps".equals(schemaName)) {
+    // apps and api is not a schema but a resource
+    if ("apps".equals(schemaName) || "api".equals(schemaName)) {
```

Comment:
could be solved by defining the routes in the 'correct' order , this feels a bit hacky